### PR TITLE
Gradle config for forked jvms

### PR DESCRIPTION
**Pull request by nik9000**
I spent 20 minutes reading gradle docs to figure out how to do this. No one
else should have to.

### CODE REVIEW

File: TESTING.asciidoc [hunk 1]
```diff
@@ -161,6 +161,11 @@ Don't count hypercores for CPU-intense tests and leave some slack
 for JVM-internal threads (like the garbage collector). Make sure there is
 enough RAM to handle child JVMs.
 
+You can set the default number of JVMs used in `~/.gradle/gradle.properties`:
```
Comment (rjernst) [inline TESTING.asciidoc]: By default this shouldn't be necessary? This is only for overriding what "auto" configures in randomized testing (usually needed b/c it rounds down, so if you have eg 6 cores like I do on a box, you get 4).

Comment (rjernst) [inline TESTING.asciidoc]: Maybe just reword so it is clear this is just for overriding the auto configured number of jvms.

**Comment by nik9000** [comment]
I'm going through my outdated PRs. Do we want this or should I just close it? I'm fine either way. I still run  the tests on 12 JVMs quite happily.

**Comment by rjernst** [comment]
LGTM

**Comment by nik9000** [comment]
PR is against the wrong branch. I'll close it and just push the changes to master.

**Comment by nik9000** [comment]
b6826bfc78bbae42e39e41189e29f6b4363c710b

**Pull request closed without merging**